.travis.yml

```diff
@@ -14,6 +14,10 @@ before_install:
 script:
   - npm run build
 after_success:
+  # add storybook to build
+  - if [[ $TRAVIS_TAG != "" || $TRAVIS_PULL_REQUEST != "false" ]]; then
```

Comment:
This will only build storybook on PRs and tagged releases. You may also want it on develop and master, like [we ended up doing](https://github.com/gnosis/dex-react/blob/develop/.travis.yml#L45).

Reply:
I don't know. What is the use case for having storybook available form master and develop?

Reply:
Just because?

Reply:
:shrug: